geoprojbase: hardcode all EPSG supported SRIDs

otan opened the pull request:
A lot of sample data sets so far that I've found support SRIDs which we
have not been able to support. So here I'm biting the bullet and loading
all EPSG SRIDs into the database. There are ~6000 of them here, so this
does take up a ~8MB of the binary space. As such, not 100% sure we
should merge this.

There were a few we were not able to support because PROJ does not like
them:
```
2020/08/05 11:30:34 error processing 29701: error from PROJ: no arguments in initialization list, skipping
2020/08/05 11:30:38 error processing 27200: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:38 error processing 27205: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:39 error processing 27206: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:39 error processing 27207: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:40 error processing 27208: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:40 error processing 27209: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:40 error processing 27210: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:41 error processing 27211: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:41 error processing 27212: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:42 error processing 27214: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:42 error processing 27215: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:43 error processing 27216: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:43 error processing 27217: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:44 error processing 27218: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:45 error processing 27219: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:45 error processing 27220: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:46 error processing 27221: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:46 error processing 27222: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:46 error processing 27223: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:47 error processing 7682: error from PROJ: unknown elliptical parameter name, skipping
2020/08/05 11:30:47 error processing 27224: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:47 error processing 27225: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:48 error processing 27226: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:48 error processing 27227: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:48 error processing 27228: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:49 error processing 27229: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:49 error processing 27230: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:50 error processing 27231: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:50 error processing 27232: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:50 error processing 27258: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:51 error processing 27259: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:51 error processing 27260: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:51 error processing 27291: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:52 error processing 27292: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:31:03 error processing 8857: error from PROJ: unknown projection id, skipping
2020/08/05 11:31:03 error processing 8858: error from PROJ: unknown projection id, skipping
2020/08/05 11:31:03 error processing 8859: error from PROJ: unknown projection id, skipping
2020/08/05 11:31:27 error processing 22300: error from PROJ: no arguments in initialization list, skipping
2020/08/05 11:31:27 error processing 6871: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:31:31 error processing 6893: error from PROJ: failed to load datum shift file, skipping
```

However, this seems to be largely NZ (r/MapsWithoutNZ) or corrupt, so
just resorted to skipping them.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/52426)
<!-- Reviewable:end -->

otan commented:
cc @petermattis (not sure quite who to ping here) - this adds about ~6000 SRIDs as a hardcoded map that gets initialized at runtime whilst adding ~8MB to the binary. it's been frustrating testing a lot of backups and items because they use some "obscure" SRID we don't yet support, and adding them one by one has been onerous. I'm not sure we can make this better (i.e. by loading them out of the database when requested) by v20.2, so curious whether you think this is ok.

otan committed:
geoprojbase: hardcode all EPSG supported SRIDs

A lot of sample data sets so far that I've found support SRIDs which we
have not been able to support. So here I'm biting the bullet and loading
all EPSG SRIDs into the database. There are ~6000 of them here, so this
does take up a ~8MB of the binary space. As such, not 100% sure we
should merge this.

There were a few we were not able to support because PROJ does not like
them:
```
2020/08/05 11:30:34 error processing 29701: error from PROJ: no arguments in initialization list, skipping
2020/08/05 11:30:38 error processing 27200: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:38 error processing 27205: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:39 error processing 27206: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:39 error processing 27207: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:40 error processing 27208: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:40 error processing 27209: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:40 error processing 27210: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:41 error processing 27211: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:41 error processing 27212: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:42 error processing 27214: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:42 error processing 27215: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:43 error processing 27216: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:43 error processing 27217: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:44 error processing 27218: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:45 error processing 27219: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:45 error processing 27220: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:46 error processing 27221: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:46 error processing 27222: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:46 error processing 27223: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:47 error processing 7682: error from PROJ: unknown elliptical parameter name, skipping
2020/08/05 11:30:47 error processing 27224: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:47 error processing 27225: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:48 error processing 27226: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:48 error processing 27227: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:48 error processing 27228: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:49 error processing 27229: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:49 error processing 27230: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:50 error processing 27231: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:50 error processing 27232: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:50 error processing 27258: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:51 error processing 27259: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:51 error processing 27260: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:51 error processing 27291: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:30:52 error processing 27292: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:31:03 error processing 8857: error from PROJ: unknown projection id, skipping
2020/08/05 11:31:03 error processing 8858: error from PROJ: unknown projection id, skipping
2020/08/05 11:31:03 error processing 8859: error from PROJ: unknown projection id, skipping
2020/08/05 11:31:27 error processing 22300: error from PROJ: no arguments in initialization list, skipping
2020/08/05 11:31:27 error processing 6871: error from PROJ: failed to load datum shift file, skipping
2020/08/05 11:31:31 error processing 6893: error from PROJ: failed to load datum shift file, skipping
```

However, this seems to be largely NZ (r/MapsWithoutNZ) or corrupt, so
just resorted to skipping them.

Release note: None

sumeerbhola reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/52426#-:-ME455qn2yil5Adf5wfs:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @sumeerbhola)



<!-- Sent from Reviewable.io -->

otan commented:
bors r=sumeerbhola
